crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__type_param.snap

```diff
@@ -0,0 +1,115 @@
+---
+source: crates/sui/src/unit_tests/upgrade_compatibility_tests.rs
+expression: normalize_path(err.to_string())
+---
+error[Compatibility E01005]: type parameter mismatch
+   ┌─ /fixtures/upgrade_errors/type_param_errors_v2/sources/UpgradeErrors.move:13:19
+   │
+13 │     public struct StructTypeParamAddU<T, U> has copy, drop { x : T }
+   │                   ^^^^^^^^^^^^^^^^^^^ Incorrect number of type parameters: expected 1, found 2
+   │
+   = Structs are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original struct's type parameters for struct 'StructTypeParamAddU' including the ordering.
+
+error[Compatibility E01005]: type parameter mismatch
+   ┌─ /fixtures/upgrade_errors/type_param_errors_v2/sources/UpgradeErrors.move:16:45
+   │
+16 │     public struct StructTypeParamRemoveCopy<T> has copy, drop { x : T }
+   │                   ------------------------- ^ Unexpected type parameter constraints: none, expected parameter with 'copy'.
+   │                   │                          
+   │                   Struct definition
+   │
+   = Structs are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original struct's type parameters for struct 'StructTypeParamRemoveCopy' including the ordering.
+
+error[Compatibility E01005]: type parameter mismatch
+   ┌─ /fixtures/upgrade_errors/type_param_errors_v2/sources/UpgradeErrors.move:19:45
+   │
+19 │     public struct StructTypeParamRemoveDrop<T: copy> has copy, drop { x : T }
+   │                   ------------------------- ^ Unexpected type parameter constraint: 'copy', expected parameter with 'copy' and 'drop'.
```

Comment:
Maybe we should print out just what is missing? Less information might be better here